### PR TITLE
clean up extract_io in train.py

### PR DESCRIPTION
**Pull request by skondrashov**
1) use one statement per line where lines wrap from being too dense
2) use a function for repeated decode_raw(substr()) calls
3) more consistent comment style 
4) align identical sections of lines
5) ignore max columns to help readability

### CODE REVIEW

File: tf/train.py [hunk 1]
```diff
@@ -69,44 +69,51 @@ def get_latest_chunks(path, num_chunks, allow_less):
...
-        1., tf.cast(bitsplat_unit_planes, tf.float32))
+    bit_planes = read(7440, 832, tf.uint8)
+    bit_planes = tf.reshape            (bit_planes, [-1, 104, 8])
+    bit_planes = tf.expand_dims        (bit_planes, -1)
+    bit_planes = tf.tile               (bit_planes, [1, 1, 1, 8])
+    bit_planes = tf.bitwise.bitwise_and(bit_planes, [128, 64, 32, 16, 8, 4, 2, 1])
+    bit_planes = tf.cast               (bit_planes, tf.float32)
+    bit_planes = tf.minimum            (bit_planes, 1.)
+
+    # Next 5 bytes are 1 or 0 to indicate 8x8 planes of 1 or 0.
```
Comment (tkondrashov) [inline tf/train.py]: my best guess, not sure if this is what the original comment was saying

File: tf/train.py [hunk 2]
```diff
@@ -69,44 +69,51 @@ def get_latest_chunks(path, num_chunks, allow_less):
...
-            tf.io.decode_raw(tf.strings.substr(raw, 8272, 5), tf.uint8), -1),
-        -1)
-    unit_planes = tf.tile(unit_planes, [1, 1, 8, 8])
-    # In order to do the conditional for frc we need to make bit unpacked versions.  Note little endian for these fields so the bitwise_and array is reversed.
-    bitsplat_unit_planes = tf.bitwise.bitwise_and(
-        unit_planes, [1, 2, 4, 8, 16, 32, 64, 128])
-    bitsplat_unit_planes = tf.minimum(
-        1., tf.cast(bitsplat_unit_planes, tf.float32))
+    bit_planes = read(7440, 832, tf.uint8)
+    bit_planes = tf.reshape            (bit_planes, [-1, 104, 8])
```
Comment (mooskagh) [inline tf/train.py]: Note that we format code with `yapf`, which does [pep8](https://www.python.org/dev/peps/pep-0008/) compliant formatting, and this change seems to be a step out of that style. (see the last paragraph of https://www.python.org/dev/peps/pep-0008/#pet-peeves)

Comment (tkondrashov) [reply tf/train.py]: yeah it does seem that way... I'll say I'm a huge proponent of using spaces to align repetitive blocks like this (you can see what it does for readability), but I'll change it to conform

**Comment by tkondrashov** [comment]
addressed @mooskagh 's comment, fixed a bug that I somehow introduced after testing the first time too

**Comment by Tilps** [comment]
I am thinking of abandoning the attempt to support multiple input formats in a single function and just have separate functions for each input format.  Which I'll do when I add input format 3.

**Comment by Tilps** [comment]
Obsolete - the code has been deleted.

**Pull request closed without merging**